app/models/grda_warehouse/hud/user_viewable_entity.rb

```diff
@@ -0,0 +1,8 @@
+module GrdaWarehouse::Hud
+
+  # NOTE, this is a join table where the users are in one database and the entities are in another
+  # there is in fact a belongs_to relationship to the user table, therefore, but it isn't specified here
+  class UserViewableEntity < Base
```

Comment:
@dfhoughton I'm not seeing a reason that this class should be within the HUD module (those classes all are specified in the HMIS data standard.)  Would it be any more difficult for this to work just in `GrdaWarehouse::UserViewableEntity`?